src/lxc/tools/lxc_unshare.c

```diff
@@ -241,12 +256,49 @@ int main(int argc, char *argv[])
 		exit(EXIT_FAILURE);
 	}
 
+	if (start_arg.setuid) {
+		start_arg.wait_fd = eventfd(0, EFD_CLOEXEC);
+		if (start_arg.wait_fd < 0) {
+			perror("eventfd");
```

Comment:
Nit: Why `perror()` instead of `fprintf()` like in all the other error paths? :)

Reply:
oops, fixed :)